storage/innobase/handler/ha_innodb.cc

```diff
@@ -473,6 +473,12 @@ ibool meb_get_checksum_algorithm_enum(const char *algo_name,
 }
 #endif /* !UNIV_HOTBACKUP */
 
+static const char *sys_tablespace_encrypt_names[] = {
+    "OFF", "ON", "RE_ENCRYPTING_TO_KEYRING", NullS};
```

Comment:
I would suggest to use "MASTER_KEY_TO_KEYRING" (or MK_TO_KEYRING) instead "RE_ENCRYPTING_TO_KEYRING"